Classes/Views/Styles.swift

```diff
@@ -19,7 +19,7 @@ enum Styles {
         static let rowSpacing: CGFloat = 8
         static let cellSpacing: CGFloat = 15
         static let tableCellHeight: CGFloat = 44
-        static let tableSectionSpacing: CGFloat = 35
+        static let tableSectionSpacing: CGFloat = 17
```

Comment:
This also effects the "no notifications" (:tada:) view. Double check that doesn't look too tight?